syntax = "proto3";

package proto_router;

import "graph.proto";

message RoutingSettings {
    int32 wait_time = 1;
    double velocity = 2;
}

message StopId {
    string name = 1;
    int32 id = 2;
}

message Router {
    RoutingSettings router_settings = 1;
    proto_graph.Graph graph = 2;
    repeated StopId stop_ids = 3;
}